src/main/webapp/data/userMapData.json

```diff
@@ -123,6 +127,7 @@
       "Talentenschool Turnhout",
       "Thomas More Vorselaar",
       "Universiteit Gent",
+      "Universiteit Hasselt",
```

Comment:
can change to `University of Hasselt`? according to google

Reply:
I was confused on this point which relates to one below: should we be putting the names as they are spelled in the language of the country they're from? Or should we translate them all to the English version of the name? I looked at a bit closer and it looks like on their dutch site they only use the name UHasselt, wikipedia says it was rebranded in 2005. The english site says Hasselt University.

Reply:
For consistency, we can use the English equivalent. I think some of the existing entries might not follow that policy, but fixing that can be a separate PR.